packages/mdc-ripple/README.md

```diff
@@ -380,6 +381,17 @@ this.ripple_ = new MDCRippleFoundation({
 });
 ```
 
+## Caveat: Edge
+
+> TL;DR ripples are disabled in Edge because of issues with its support of CSS variables in pseudo elements.
+
+Edge introduced CSS variables in version 15.  Unfortunately, there are
+[issues](https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/11495448/)
+involving its implementation for `::before` and `::after` pseudo-element selectors which cause ripples to
+behave incorrectly.  We feature-detect one of these bugs and do not upgrade ripples if the bug is observed.
```

Comment:
"issues" was based on the wording in the linked report. I'm updating the wording to hopefully resolve the ambiguity though.